superbench/benchmarks/micro_benchmarks/hipblaslt_function.py

```diff
@@ -97,9 +124,7 @@ def _process_raw_result(self, cmd_idx, raw_output):
             fields = lines[index + 1].strip().split(',')
 
             # Check the number of fields and the format of the first two fields
```

Comment:
Why we don't check format of first two fields anymore?

Reply:
since I found the fp8 output is 1.xxe6, so my original check is incorrect